spmd/compiler/fusion.py

```diff
@@ -32,25 +37,45 @@ class FusionElement:
     """This class tracks the nodes for a DTensor expanded communication collective in the graph"""
 
     in_graph: bool = False
+    processed: bool = False
+    size: Optional[int] = 0
+    shape: Optional[Iterable[[int], [int]]] = field(default_factory=lambda: [])  # type: ignore
     comm_type: Optional[CommType] = None
     node_list: Optional[List[fx.Node]] = field(default_factory=lambda: [])  # type: ignore
-    size: int = 0
     prev_node: Optional[fx.Node] = None  # node that was before start of section
-    next_node: Optional[fx.Node] = None  # node that was after end of section
-    processed: bool = False
     output_name: str = ""
-    clone_node: Optional[fx.Node] = None
-    comm_node: Optional[fx.Node] = None
-    wait_node: Optional[fx.Node] = None
+    comm_node: Optional[fx.Node] = None  # type: ignore
+    wait_node: Optional[fx.Node] = None  # type: ignore
+    grad_tensor_node: Optional[fx.Node] = None  # type: ignore
+
+    def _get_next_node(
+        self,
+    ) -> fx.Node:
+        """get the next node after this FE section"""
```

Comment:
nit: Can you capitalize the first letter in your doc comments?

Reply:
Yes - good idea.  I've updated all doc strings for current and will continue with that style going forward.